packages/ui/components/QuestionFormInput/index.tsx

```
@@ -634,13 +727,21 @@ export const QuestionFormInput = ({
             attributeClasses={attributeClasses}
+            stripHtmlTags={stripHtmlTags}
           />
         )}
       </div>
+
       {usedLanguageCode !== "default" && value && typeof value["default"] !== undefined && (
         <div className="mt-1 text-xs text-gray-500">
           <strong>Translate:</strong>{" "}
-          {recallToHeadline(value, localSurvey, false, "default", attributeClasses)["default"]}
+          <label
+            dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)